### PR TITLE
Add additional gen information for zinv photon studies

### PR DESCRIPTION
**Pull request by pastika**
Change contingent on my checking effects on file size

### CODE REVIEW

File: python/processors/Stop0l_postproc.py [hunk 1]
```diff
@@ -266,7 +266,7 @@ def main(args):
             BtagSFWeightProducer("allInOne_bTagEff_deepCSVb_med.root", args.sampleName, DeepCSVMediumWP[args.era]),
             # statusFlag 0x2100 corresponds to "isLastCopy and fromHardProcess"
             # statusFlag 0x2080 corresponds to "IsLastCopy and isHardProcess"
-            GenPartFilter(statusFlags = [0x2100, 0x2080, 0x2000], pdgIds = [0, 0, 22], statuses = [0, 0, 1]),
+            GenPartFilter(statusFlags = [0x2100, 0x2080, 0x2000, 0], pdgIds = [0, 0, 22, 0], statuses = [0, 0, 1, 23]),
```
Comment (zhenbinwu) [inline python/processors/Stop0l_postproc.py]: What are these genpart? Can you provide more information?

**Comment by pastika** [comment]
increases file size by around 1.5%

**Comment by zhenbinwu** [comment]
That is Ok. But I want to understand what are added and whether others should revisit their code.

**Comment by pastika** [comment]
All this does is add status==23 gen particles.  These are the particles which are labeled "outgoing from hard process".  There should be no need for anyone to change anything, so long as they are already selecting the status codes or statusFlags they require.

**Comment by zhenbinwu** [comment]
ok, merging this PR

**Pull request merged**